index.js

```diff
@@ -45,13 +48,17 @@ function checkRequiredInput(inputName, inputValue) {
 async function executeShellScript(scriptName, ...parameters) {
     parameters = (parameters || []).join(' ');
     const command = `docker_buildx/scripts/${scriptName}.sh ${parameters}`;
-    child_process.execSync(command, {stdio: 'inherit'});
+    child_process.execSync(command, { stdio: 'inherit' });
 }
 
 async function buildAndPublish(platform, imageName, imageTag, dockerFile, buildArg, load, context, target) {
-    const dockerHubUser = extractInput('dockerHubUser', true);
-    const dockerHubPassword = extractInput('dockerHubPassword', true);
-    await executeShellScript('dockerhub_login', dockerHubUser, dockerHubPassword);
+    const dockerUser = extractInput('dockerUser', false, extractInput('dockerHubUser', false));
+    checkRequiredInput('dockerUser (or dockerHubUser)', dockerUser);
```

Comment:
I think that you can avoid this force check and enhance the correlation between the fields.
For example:
const dockerHubUser = extractInput('dockerHubUser', false);
const dockerUser = extractInput('dockerUser', !dockerHubUser, dockerHubUser);

In this way, we check that dockerUser is required only when dockerHubUser isn't defined and suggest to the user to define the new parameter (instead of the previous one)

Same for the password